Modules/cli_manager.py

```diff
@@ -16,40 +16,29 @@
 """
 
 import argparse
-import logging
 
 # create a new parser
 _parser = argparse.ArgumentParser()
 
-# add arguments
-# - I would wrap this in a list of tuples to be looped over, but each argument may is too specific
-# - to enable looping. So we will register them manually.
-
 # register optional flag for --clean-log, truncating existing log file and starting a new blank one.
-_parser.add_argument("--clean-log", help="Start with a clean log file.  "
-                                         "Deletes any existing log data.",
+_parser.add_argument("--clean-log",
+                     help="Start with a clean log file. Deletes any existing log data.",
                      action="store_true")
+
 # register optional argument for the config file
-_parser.add_argument("--config-file", "--config", help="Specify the configuration file to load, "
-                                                       "relative to config/",
+_parser.add_argument("--config-file", "--config", "-c",
```

Comment:
i don't agree with `-c` being a valid shorthand for this cli argument. imho single char arguments should be flags.

Reply:
In terms of convention, there shouldn't really be a difference between regular options and 'long' options `--` in what they are used for. [The POSIX specification added options (`-o`), while the GNU specification added long options (`--option`) as a more easily understandable version of the same thing.](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html)
Also, only one out of three of the flags we have has a short option, so we aren't really on that route anyways.
Interestingly, GNU wants long options with arguments to be referenced by `--option=arg`, but I don't think argparse supports this.

Reply:
grumbles